backend/mailman_lambda.py

```diff
@@ -27,7 +28,9 @@ def lambda_handler(event, context):
             )
```

Comment:
Does this logic assume that we will always have a subscription?

Reply:
@rogebrd Yes it does. I was planning to go back and add error handling once we had everything working end to end.